compute/functions/reference-content/code-examples.mdx

```diff
@@ -450,6 +452,45 @@ Please refer to our guide on [code packaging](https://www.scaleway.com/en/docs/c
   };
   ```
 
+
+### Using ES Modules
+
+[Official ESM documentaton](https://nodejs.org/api/esm.html)
+
+ES-Modules are available since `node16` runtime on Scaleway Serverless Functions.
```

Comment:
```suggestion
Node has two module systems: `CommonJS` modules and `ECMAScript` (`ES`) modules. By default, Node treats your code files as CommonJS modules, however [ES modules](https://nodejs.org/api/esm.html) have also been available since the release of `node16` runtime on Scaleway Serverless Functions. ES modules give you a more modern way to re-use your code.
```